typings/vscode-mssql.d.ts

```diff
@@ -178,6 +178,22 @@ declare module 'vscode-mssql' {
 		osVersion: string;
 	}
 
+	/**
+	 * The possible values of the server engine edition
```

Comment:
I'd also add a link to https://docs.microsoft.com/sql/t-sql/functions/serverproperty-transact-sql and direct them to look at the `EngineEdition` property